apps/src/package.json

```diff
@@ -6,7 +6,7 @@
   "browserify": {
     "transform": [
       "ejsify",
-      "reactify"
+      ["babelify", {"compact": false}]
```

Comment:
the `compact=false` setting is needed to silence warnings Babel generates on large files, which we have a few of in our codebase:
`>> [BABEL] Note: The code generator has deoptimised the styling of "[path]/code-dot-org/apps/build/js/studio/studio.js" as it exceeds the max of "100KB".`